server/views/temporary-accommodation/bedspaces/show.njk

```diff
@@ -39,7 +39,18 @@
     items: actions
   }) }}
 
-  {{ locationHeader({ premises: premises }) }}
+  <div class="govuk-grid-row">
```

Comment:
I'd suggest something like:
```
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-one-third">
      {{ locationHeader({ premises: premises }) }}
    </div>
    <div class="govuk-grid-column-one-third">
      <div class="attributes-header details">
        <h2>Property attributes</h2>
        <ul>
          {% for characteristic in premisesCharacteristics %}
          <li>{{ characteristic }}</li>
          {% endfor %}
        </ul>
      </div>
    </div>
  </div>
```
and define `attributes-header` similar to `location-header`

So:
* Fix up the indentation
* Keep the "layout" divs separate from the "content" divs (and follow a similar hierarchy to the location header)
* Put the <li> inside a <ul>

My instinct would be to prefer having duplication between an `attributes-header` and `location-header`, than muddle what the `location-header` class is for and need to look for either the first or second occurrence of the class (though the duplication is work-round-able with SCSS [@includes](https://sass-lang.com/documentation/at-rules/mixin)).

Or maybe rename the CSS class `location-header` to something more generic, and put `id="location-header"` in the Location Header macro, `id="attributes-header"` in the attributes header, and use those in the integration test to actually find the thing you're testing

Reply:
done - thanks!